.gitignore

```diff
@@ -57,7 +57,7 @@ coverage.xml
 
 # Django stuff:
 *.log
-local_settings.py
+local_settings*.py
```

Comment:
I'm using InteliJ 2024.2.3 which seems to favour providing local settings with a Python file.

I have numerous different configurations and this change therefore supports multiple local files.

Reply:
You can use instead your own user side `~/.gitignore`.

Reply:
Agreed; however since `local_settings.py` was already _ignored_ I just expanded the filter.